2021q1/clusteradm.adoc

```diff
@@ -1,45 +1,45 @@
-## Cluster Administration Team ##
+== Cluster Administration Team
 
 Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
 
-Link: link:https://www.freebsd.org/administration.html#t-clusteradm[Cluster Administration Team members]
+Link: link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members]
 
 The FreeBSD Cluster Administration Team consists of the people responsible for administering the machines that the Project relies on for its distributed work and communications to be synchronised. In this quarter, the team has worked on the following:
 
-  * Installed a new package builder
-  * Added Git support to cluster management scripts
-  * Put local Git mirrors on the universe machines
-  * Replaced disks in the UK mirror
-  * Replaced a disk in pointyhat (https://pkg-status.freebsd.org)
-  * Recycled some old dead-weight servers eating up rackspace and power at our primary cluster site
-  * Upgraded developer reference platforms
-    * ref{11,12,13,14}-{amd64,i386}
-    * universe*
-  * Installed two new aarch64 machines
-    * ref12-aarch64, ref13-aarch64, ref14-aarch64
-    * security-officer aarch64 freebsd-update builder
```

Comment:
Same with these entries as the ref and universe machines, these are also sub-items of the aarch64 machines.